cli/objects/rule.py

```diff
@@ -198,31 +192,28 @@ def _get_jira_component(self, rule_dict: dict[Any, Any]) -> Optional[list[str]]:
             Optional[list[str]]: A list of strings representing the component(s) defined in a firewatch rule. If not defined, return None
         """
         components = []
-        if "jira_component" in rule_dict.keys():
-
-            jira_component = rule_dict["jira_component"]
-
-            if isinstance(jira_component, str):
-                components.append(jira_component)
-                return components
-            elif isinstance(jira_component, list):
-                for component in jira_component:
-                    if isinstance(component, str):
-                        components.append(component)
-                    else:
-                        self.logger.error(
-                            f'Component "{component}" in "jira_component" is not a string in firewatch rule: "{rule_dict}"',
-                        )
-                        exit(1)
-                return components
-            else:
-                self.logger.error(
-                    f'Value for "jira_component" must be either a list of strings (multiple components) or a string value (single component) in firewatch rule: "{rule_dict}"',
-                )
-                exit(1)
-
+        jira_component = rule_dict.get("jira_component")
+
+        if isinstance(jira_component, str):
```

Comment:
way not force `jira_component` to be always list?
It's better to keep the format (for the code and also for the user experience)

Reply:
Because there are already configs in OpenShift CI using the string format. I agree with you, but getting them to switch it over is going to be difficult.

Reply:
I don't think this change is going to be made, at least not in this PR, unfortunetly. We can make this change in a v2 version of the tool so that we don't break people using this feature already.

Reply:
make sense, can you open an issue for this?

Reply:
@myakove yes [here it is](https://github.com/CSPI-QE/firewatch/issues/52#issue-1875528773)